.github/workflows/main.yml

```diff
@@ -1083,6 +1083,9 @@ jobs:
       - name: platform/options.api
         run: ant $OPTS -Dvanilla.javac.exists=true -f platform/options.api test-unit
 
+      - name: platform/options.editor
+        run: ant $OPTS -Dvanilla.javac.exists=true -f platform/options.editor test-unit
+
```

Comment:
oops. This should be `ide/options.editor` and should be moved to the `ide-modules-test` job.